l5kit/l5kit/rasterization/rasterizer_builder.py

```diff
@@ -65,9 +73,27 @@ def build_rasterizer(cfg: dict, data_manager: DataManager) -> Rasterizer:
     history_num_frames = cfg["model_params"]["history_num_frames"]
 
     if map_type in ["py_satellite", "satellite_rgb"]:
-        sat_image, meta = _load_image_and_metadata(raster_cfg["satellite_map_key"], data_manager)
-        ecef_to_sat = np.array(meta["ecef_to_image"], dtype=np.float64)
-        pose_to_ecef = load_pose_to_ecef()
+        sat_image_key = raster_cfg["satellite_map_key"]
+        sat_image = _load_satellite_map(sat_image_key, data_manager)
+
+        sat_meta_key = os.path.splitext(sat_image_key)[0] + ".json"
+        ecef_to_sat = np.array(_load_metadata(sat_meta_key, data_manager)["ecef_to_image"], dtype=np.float64)
+        try:
+            pose_to_ecef = np.array(_load_metadata(data_meta_key, data_manager)["pose_to_ecef"], dtype=np.float64)
```

Comment:
Why it could fail? Where the meta data file is stored, I can not find it in S3 `l5kit` folder

Reply:
because even if we upload it now, it will be shipped only with the new dataset release to users. So in the meantime we fall back to the original hard-coded matrix